rpc.go

```diff
@@ -113,6 +114,28 @@ type libvirtError struct {
 	Level    uint32
 }
 
+func (e libvirtError) Error() string {
+	return fmt.Sprintf("%s, code: %d, domain id: %d", e.Message, e.Code, e.DomainID)
+}
+
+func checkError(err error, expectedError ErrorNumber) bool {
+	e, ok := err.(libvirtError)
+	if ok {
+		return e.Code == uint32(expectedError)
+	}
+	return false
+}
+
+// ErrIsNotFound detects libvirt's ERR_NO_DOMAIN.
+func ErrIsNotFound(err error) bool {
+	return checkError(err, ErrNoDomain)
+}
+
+// ErrIsOK detects libvirt's ERR_OK.
```

Comment:
This shouldn't exist - if libvirt returns ERR_OK, the Go wrapper functions should always return a nil error. It'd be poor go style to have to call an error classifier to determine whether there's an error at all.